src/test/java/vention/pagesTest/LoginFlowTest.java

```diff
@@ -0,0 +1,49 @@
+package vention.pagesTest;
+
+import vention.configLoader.ConfigLoader;
+import vention.driver.DriverManager;
+import vention.pages.CookiePopupPage;
+import vention.pages.PreLoginPage;
+import vention.pages.LoginPage;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+
+public class LoginFlowTest extends BaseTest {
+  private CookiePopupPage cookiePopupPage;
+  private PreLoginPage preLoginPage;
+  private LoginPage loginPage;
+
+  @BeforeMethod
+  public void beforeMethod() {
+    String url = ConfigLoader.get("pre-login-url");
```

Comment:
WHy do you have many urls? How are you going to maintain them later? in case you need to add a new test for a new page you will add a new url?

App has one common URL but URIs might be different and they are related to Pages (Page Objects in terms of automation, you can create a method 'openPage' which is common for all pages and navigate to it via that method)